src/comparisons/events/alternatives.txt

```diff
@@ -1 +1,3 @@
 ftdomdelegate: https://github.com/ftlabs/ftdomdelegate
+defer: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer
+modules: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules
```

Comment:
Why are they listed as alternatives for events?

Reply:
They're in the same section as the document ready example, and I wanted to give some alternatives, but I can't type HTML in the comparisons yet

Reply:
Hmm, ok. Maybe we should open an issue for that?

Reply:
It should be easier to fix when the Astro port is done. I don’t think it’s worth the effort to fix that separately yet.

Reply:
That's fair. But do you want to open an issue anyway and mention that it should wait for the port to Astro to be done?

Reply:
You mean as a reminder rather than a blocker? I'm fine with that.